examples/Simple Vertically Partitioned SplitNN.ipynb

```diff
@@ -136,12 +135,14 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Data preprocessing\n",
-    "transform = transforms.Compose([transforms.ToTensor(),\n",
-    "                              transforms.Normalize((0.5,), (0.5,)),\n",
-    "                              ])\n",
-    "trainset = datasets.MNIST('mnist', download=True, train=True, transform=transform)\n",
-    "trainloader = torch.utils.data.DataLoader(trainset, batch_size=64, shuffle=True)"
+    "# Create dataset\n",
+    "data = add_ids(MNIST)(\".\", download=True, transform=ToTensor())  # add_ids adds unique IDs to data points\n",
+    "\n",
+    "# Split data\n",
+    "data_partition1, data_partition2 = partition_dataset(data, remove_data=False)\n",
```

Comment:
the `partition_dataset` function will need the parameter `keep_order=True` (False by default) until PSI is implemented